chart/dockerfiles/pgbouncer-exporter/build_and_push.sh

```diff
@@ -21,13 +21,13 @@ readonly DOCKERHUB_USER
 DOCKERHUB_REPO=${DOCKERHUB_REPO:="airflow"}
 readonly DOCKERHUB_REPO
 
-PGBOUNCER_EXPORTER_VERSION="0.5.0"
+PGBOUNCER_EXPORTER_VERSION="0.13.0"
 readonly PGBOUNCER_EXPORTER_VERSION
 
-AIRFLOW_PGBOUNCER_EXPORTER_VERSION="2021.04.28"
+AIRFLOW_PGBOUNCER_EXPORTER_VERSION="2021.10.29"
 readonly AIRFLOW_PGBOUNCER_EXPORTER_VERSION
 
-EXPECTED_GO_VERSION="1.15.1"
+EXPECTED_GO_VERSION="1.17"
 readonly EXPECTED_GO_VERSION
 
 COMMIT_SHA=$(git rev-parse HEAD)
```

Comment:
Not PR related to that, but why not use [a multi-stage build](https://docs.docker.com/develop/develop-images/multistage-build/) if we need two images to build?

Reply:
I created a PR: https://github.com/apache/airflow/pull/19327